parts/linux/cloud-init/artifacts/cse_helpers.sh

```diff
@@ -99,6 +99,7 @@ ERR_K8S_INSTALL_ERR=204 # Error installing or setting up kubernetes binaries on
 ERR_SYSTEMCTL_MASK_FAIL=2 # Service could not be masked by systemctl
 
 ERR_CREDENTIAL_PROVIDER_DOWNLOAD_TIMEOUT=205 # Timeout waiting for credential provider downloads
+ERR_CREDENTIAL_PROVIDER_CONFIG=206 # Error checking credential provider flags
```

Comment:
suggest to use a separate pr to check in linux error codes

Reply:
removed